### DISCUSSION TITLE
Postgress ILIKE support.

### DISCUSSION THREAD

**Discussion by studentenherz** [SeaQL/sea-orm#1689] [Q&A] [answered]
I've searched the repo and don't see support for PostgreSQL's `ILIKE`. If `LIKE` is implemented, could `ILIKE` be easily implemented too?

Maybe because I'm relatively new to Rust or because the repo is complex (or maybe both) I don't know how to do it myself.

Right now I'm relying on `from_raw_sql`.

**Comment by billy1624** [comment, marked as answer]
Hey @studentenherz, we do have `ILIKE`. https://docs.rs/sea-query/latest/sea_query/extension/postgres/trait.PgExpr.html#method.ilike

**Reply by studentenherz** [reply]
Thanks! I did't find that when I looked up before.

Another question: How can I use that in something like this:

```rust
Dle::find()
    .filter(Column::Lemma.ilike(format!("{}%", query)))
    .limit(10)
    .all(&self.db)
    .await
    .unwrap()
```

? Do I need to do something with my model?

**Reply by billy1624** [reply]
Try this
```rs
Dle::find()
    .filter(Expr::col(Column::Lemma).ilike(format!("{}%", query)))
    .limit(10)
    .all(&self.db)
    .await
    .unwrap()
```

**Reply by joebnb** [reply]
im stucked on this,for searching how to support ilike,curisely why not expose it on sea-orm,may this will help user avoid to search it
